demo/popover/index.html

```diff
@@ -12,6 +12,8 @@
     <script src="../app.js"></script>  
 
     <script>
+      var app = angular.module('myApp', ['onsen']);
```

Comment:
I think you should do `angular.module('myApp')` since the app is already initialized.